runtime/kusama/src/lib.rs

```diff
@@ -1525,7 +1525,12 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
-	(RemoveCollectiveFlip, MigratePalletVersionToStorageVersion),
+	(
+		RemoveCollectiveFlip,
+		MigratePalletVersionToStorageVersion,
+		CouncilStoragePrefixMigration,
+		TechnicalCommitteeStoragePrefixMigration
```

Comment:
I mixed up the merge, those migration are incompatible, the latter migration are checking pallet version, while the former is writing the new version into storage.